src/tello/tello.py

```diff
@@ -488,3 +488,57 @@ def move_up(self, distance) -> str:
         """
 
         return self.move('up', distance)
+
+    def go_xyz_speed(self, x: int, y: int, z: int, speed: int):
+        """Fly to x y z relative to the current position.
+        Speed defines the traveling speed in cm/s.
+        'x', 'y' and 'z' values can't be set between -20 - 20 simultaneously
+        Arguments:
+            x: -500 - 500
+            y: -500 - 500
+            z: -500 - 500
+            speed: 10 - 100
+
+        """
+
+        min_movement = 20
+        max_movement = 500
+        min_speed = 10
+        max_speed = 100
```

Comment:
This should be defined at the top of the class, before init. And constants should be UPPER_CASE